tox.ini

```diff
@@ -52,11 +171,16 @@ exclude =
     */migrations/*.py,
     .eggs/*,
 
-[testenv:lint]
-# Invoke with: tox -e lint
-# This should run identically to tox -e lint-py36
-recreate=False
-basepython=python3.6
-envdir={toxworkdir}/lint-py36
-deps={[lint]deps}
-commands={[lint]commands}
+[isort]
```

Comment:
Right now in this repo, the `isort` configuration is in its own file, https://github.com/cfpb/teachers-digital-platform/blob/master/.isort.cfg. If we want to move it here, that's fine (and where we standardized), but we should remove that file.

Reply:
Removed `.isort.cfg` and updated isort in `tox.ini`